marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/loadDataSettings.mjs

```diff
@@ -0,0 +1,68 @@
+/**
+ Copyright 2012-2019 MarkLogic Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+'use strict';
+
+const DataHubSingleton = require('/data-hub/5/datahub-singleton.sjs');
+
+// define constants for caching expensive operations
+const dataHub = DataHubSingleton.instance();
+
+const collections = ['http://marklogic.com/data-hub/load-data-artifact/settings'];
+const databases = [dataHub.config.STAGINGDATABASE, dataHub.config.FINALDATABASE];
+const permissions = [xdmp.permission(dataHub.consts.DATA_HUB_DEVELOPER_ROLE, 'update'), xdmp.permission(dataHub.consts.DATA_HUB_OPERATOR_ROLE, 'read')];
```

Comment:
@ryanjdew ,
Should the permissions here be "data-hub-load-data-reader,read,data-hub-load-data-reader,update" ? If so, this change has to be made for all the artifacts ?

Reply:
I believe @bsrikan will be addressing that in another PR (https://github.com/marklogic/marklogic-data-hub/pull/3536). If not, we can create a task to adjust it after the roles have been added.